src/components/resource-section/add-evaluation/AddEvaluation.js

```diff
@@ -28,17 +26,16 @@ const AddEvaluation = ({ formOptions }) => {
         value={evaluation}
         onChange={(e) => setEvaluation(e.value)}
         cancel={false}
-        readOnly={evaluated}
+        readOnly={formOptions.evaluated}
         className={styles.inputRating}
       />
       <InputTextarea
         rows={4}
-        cols={80}
-        value={comment}
+        cols={15}
+        value={comment || ''}
         onChange={(e) => setComment(e.target.value)}
-        disabled={evaluated}
+        disabled={formOptions.evaluated}
```

Comment:
The InputTextArea as well as the InputRating show as disabled after entering a Resource without any evaluations ("Sin Evaluacion") and going out. Then, even without evaluation, the Inputs stays disabled, except from the first resource.